components/prize_pool/commons/prize_pool.lua

```diff
@@ -194,9 +204,16 @@ function PrizePool:create()
 		self:addPrize(PRIZE_TYPE_USD, 1)
 	end
 
+	table.sort(self.prizes, PrizePool._sortPrizes)
+
 	return self
 end
 
+--- Sort function for Prizes based on the lowest sortOrder
+function PrizePool._sortPrizes(x, y)
```

Comment:
```suggestion
function PrizePool._comparePrizes(x, y)
```

Bit more common for this functionality I believe (only for the iterator function)